moleculerize/__init__.py

```diff
@@ -80,7 +80,7 @@ def generate_hosts_inventory(json_inventory):
     return inventory_hosts
 
 
-def render_molecule_template(inventory_hosts, template_file):
+def render_molecule_template(scenario, inventory_hosts, template_file):
```

Comment:
Can you please update the docstring to match the new function signature.

Reply:
Docstring and tests updated. Thanks @zreichert